components/filesystem/introduction.rst

```diff
@@ -234,6 +234,20 @@ isAbsolutePath
     // return false
     $fs->isAbsolutePath('../dir');
 
+tempNam
+~~~~~~~
+
+.. versionadded:: 2.7
+    The ``tempnam()`` function was introduced in Symfony 2.7.
+
+:method:`Symfony\\Component\\Filesystem\\Filesystem::tempnam` is a drop-in
+replacement for PHP's builtin tempnam which allows you to create a temporary
```

Comment:
I would use the `phpfunction` role here to link to the documentation of the PHP `tempnam()` function:

```rst
[...] PHP's built-in :phpfunction:`tempnam` function which [...]
```